test/burn/org/apache/cassandra/concurrent/LongSharedExecutorPoolTest.java

```diff
@@ -135,19 +139,27 @@ private void testPromptnessOfExecution(long intervalNanos, float loadIncrement)
         {
             if (System.nanoTime() > until)
             {
-                System.out.println(String.format("Completed %.0fK batches with %.1fM events", runs * 0.001f, events * 0.000001f));
+                logger.info(String.format("Completed %.0fK batches with %.1fM events", runs * 0.001f, events * 0.000001f));
                 events = 0;
                 until = System.nanoTime() + intervalNanos;
                 multiplier += loadIncrement;
-                System.out.println(String.format("Running for %ds with load multiplier %.1f", TimeUnit.NANOSECONDS.toSeconds(intervalNanos), multiplier));
+                logger.info(String.format("Running for %ds with load multiplier %.1f", TimeUnit.NANOSECONDS.toSeconds(intervalNanos), multiplier));
             }
 
             // wait a random amount of time so we submit new tasks in various stages of
             long timeout;
+            boolean timeoutIsMax = false;
```

Comment:
why did you keep `timeout` and not renamed it to `deadline`?

Reply:
to keep changes minimal, but I'll consider changing it